frontend/components/ResourceInfoCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Grid, Paper, Sx, Text, rem } from '@mantine/core';
+import Link from 'next/link';
+import React from 'react';
+import { ResourceInfo } from '@/util/types/fhir';
+import { ArrowNarrowRight } from 'tabler-icons-react';
+
+export interface ResourceInfoCardProps {
+  resourceInfo: ResourceInfo;
+  selected?: boolean;
+}
+
+export default function PatientInfoCard({ resourceInfo, selected }: ResourceInfoCardProps) {
```

Comment:
I have a lot of UI/UX paradigms/thoughts on this component. I'll list them out here, but let me know if you want to talk about any of them more:

1. I think there's no need to have a "collapsed" version that gets expanded later, I think the info should just be expanded always
2. Can we style the "View" button a little differently? It takes up a lot of space in the middle of the card. What if it was just some type of "expand icon" button in the top right or something, indicating that it will take you to a new page
3. This is the biggest one. It's very common when you start out rendering structured data to default to the approach of rendering `Key: Value`, as it would appear on an object.

We have a lot more freedom in a frontend app for how we render data. Take a look at the following example. Is my version intuitive enough that a user looking at it might know what each of these pieces of data is intuitively without needing a key value rendering? Maybe. I think we should push for less labels somehow if possible.

<img width="982" alt="image" src="https://user-images.githubusercontent.com/16297930/236026641-8cdb81b2-7927-4abf-a9cb-2821e6f726b4.png">

<img width="762" alt="image" src="https://user-images.githubusercontent.com/16297930/236026253-f24d3589-8a1f-4d20-9a1d-6663a23e9b8a.png">